apps/onnx/Makefile

```diff
@@ -76,11 +76,11 @@ test: $(BIN)/$(HL_TARGET)/onnx_converter_test $(BIN)/$(HL_TARGET)/onnx_converter
 	LD_LIBRARY_PATH=$(BIN) $(BIN)/$(HL_TARGET)/onnx_converter_test
 	LD_LIBRARY_PATH=$(BIN) $(BIN)/$(HL_TARGET)/onnx_converter_generator_test
 
-PYTHON ?= python3
-PYBIND11_CFLAGS = $(shell $(PYTHON) -m pybind11 --includes) -frtti
+PYTHON ?= python
+PYBIND11_CFLAGS = $(shell $(PYTHON)-config --includes) -frtti -undefined dynamic_lookup 
```

Comment:
"-undefined dynamic_lookup" is probably only going to compile on OSX.

Reply:
Indeed,  this should be fixed now